lnplus/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LNPLUS_IP
+      APP_PORT: $APP_LNPLUS_PORT
+  web:
+    image: lightningnetworkplus/lnpclient:v0.1.10
```

Comment:
the image is missing the sha256 after the version information.

Reply:
Thanks! I now added it.